docs/tutorial.md

```diff
@@ -73,35 +75,39 @@ Here we're listening for [`chat`](http://mineflayer.prismarine.js.org/#/api?id=q
 [`chat`](http://mineflayer.prismarine.js.org/#/api?id=quotchatquot-username-message-translate-jsonmsg-matches) event is only emitted when a player chats publicly.
 
 ```js
-function echo(username, message) {
-  if (username === bot.username) return;
-  bot.chat(message);
+function echo (username, message) {
+  if (username !== bot.username) {
+    bot.chat(message)
+  }
 }
 
-bot.on("chat", echo); // Only emitted when a player chats publicly.
+bot.on('chat', echo) // Only emitted when a player chats publicly.
 ```
 
-`bot.on('chat', echo)` will listen for [`chat`](http://mineflayer.prismarine.js.org/#/api?id=quotchatquot-username-message-translate-jsonmsg-matches) event, and execute `echo` function with arguments `username`, `message`, `translate`, `jsonMsg`, and `matches` when the event emitted.
+`bot.on('chat', echo)` - will listen for [`chat`](http://mineflayer.prismarine.js.org/#/api?id=quotchatquot-username-message-translate-jsonmsg-matches) event, and execute `echo` function with arguments `username`, `message`, `translate`, `jsonMsg`, and `matches` when the event emitted.
 
 #### Welcome Bot
 
 Here we're listening for [`playerJoined`](http://mineflayer.prismarine.js.org/#/api?id=quotplayerjoinedquot-player) event.
 And sending welcome message to chat.
 
 ```js
-function joined(player) {
-  bot.chat(`Welcome ${player}! :D`);
+function joined (player) {
+  bot.chat(`Welcome ${player}! :D`)
 }
 
-bot.on("playerJoined", joined);
+bot.on('playerJoined', joined)
 ```
 
 ### Callbacks
 
-Callback is a function as argument to the main function, when the main function finish or got an error then the callback executed.
+Callback (known as a "call-after" function) is a function that is passed as an argument to the other code / function.
+If that other code / function finish the execution, then the callback is called.
```

Comment:
I'd rather say that some functions like `bot.fish()` or any other trivial example will take some time before finishing so that's why the callbacks are used.